app/models/topic_link.rb

```diff
@@ -58,8 +58,7 @@ def self.topic_map(guardian, topic_id)
 
     builder.where('ftl.topic_id = :topic_id', topic_id: topic_id)
     builder.where('ft.deleted_at IS NULL')
-    # note that ILIKE means "case insensitive LIKE"
-    builder.where("NOT(ftl.url ILIKE '%.png' OR ftl.url ILIKE '%.jpg' OR ftl.url ILIKE '%.gif')")
+    builder.where("ftl.extension NOT IN ('png','jpg','gif')")
```

Comment:
Note that I'm not making any opinions about whether this extension list is complete as the extensions we use here has been added 5 years ago. 

If I were to give my opinion... I think there may be other file type extensions we may want to exclude...